packages/plugin-core/src/commands/NoteLookupCommand.ts

```diff
@@ -557,7 +555,9 @@ export class NoteLookupCommand
           targetNote: nodeNew,
           engine,
         });
-        AnalyticsUtils.track(EngagementEvents.TemplateApplied);
+        AnalyticsUtils.track(EngagementEvents.TemplateApplied, {
+          source: this.key,
```

Comment:
I think it'll be clearer for us when looking at telem to have this property called 'SchemaTemplate' instead of 'NoteLookup'?

Reply:
a schema template can be applied via `NoteLookup` or `GotoNote`. essentially, a template will always be a "schema template" unless it is called via `TemplateApply`

Reply:
But do we actually care if it's `NoteLookup` vs `GoToNote`? Seems like we should only care if it's automatic application (schema template) vs manual application (apply template command)

Anyway - I'm fine with it, let's just add notes in Airtable about how to interpret what a value with 'NoteLookup' means.